packages/cli/src/parallel-tests/run-all-tests.ts

```diff
@@ -213,59 +220,22 @@ export const runAllTests = async ({
     await storeTestRunResults("Running", resultsSoFar);
   };
 
-  const getResults = async () => {
-    if (parallelTasks == null || parallelTasks > 1) {
-      const results = await runAllTestsInParallel({
-        config,
-        testRun,
-        testsToRun,
-        executionOptions,
-        screenshottingOptions,
-        apiToken,
-        commitSha,
-        appUrl,
-        useAssetsSnapshottedInBaseSimulation,
-        parallelTasks,
-        deflake,
-        replayEventsDependencies,
-        onTestFinished,
-      });
-      return results;
-    }
-
-    const results: DetailedTestCaseResult[] = [];
-    const progress: TestRunProgress = {
-      runningTestCases: testsToRun.length,
-      failedTestCases: 0,
-      passedTestCases: 0,
-    };
-    for (const testCase of testsToRun) {
```

Comment:
I deleted the code to run in serial, since it's essentially the same as running in parallel with parallelTasks equal to 1, and the fewer code paths to support/environmental differences the better.